mmv1/products/spanner/InstanceConfig.yaml

```
@@ -11,23 +11,110 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
 --- !ruby/object:Api::Resource
-name: 'InstanceConfig'
-base_url: 'projects/{{project}}/instanceConfigs'
+name: InstanceConfig
+base_url: projects/{{project}}/instanceConfigs
+update_mask: true
+update_verb: :PATCH
 description: |
   A possible configuration for a Cloud Spanner instance. Configurations
   define the geographic placement of nodes and their replication.
-readonly: true
-# Used as a resource ref
-exclude: true
+references: !ruby/object:Api::Resource::ReferenceLinks
+  guides:
+    Official Documentation: https://cloud.google.com/spanner/
+  api: https://cloud.google.com/spanner/docs/reference/rest/v1/projects.instanceConfigs
+async: !ruby/object:Api::OpAsync
+  actions: [create, update]
+  operation: !ruby/object:Api::OpAsync::Operation
+    base_url: '{{op_id}}'
+  result: !ruby/object:Api::OpAsync::Result
+    resource_inside_response: true
+autogen_async: true
+id_format: '{{project}}/{{name}}'
+import_format:
+  - projects/{{project}}/instanceConfigs/{{name}}
+  - '{{project}}/{{name}}'
+  - '{{name}}'
+examples:
+  - !ruby/object:Provider::Terraform::Examples
+    name: spanner_instance_config_basic
+    primary_resource_id: example
+    vars:
+      instance_config_name: '"custom-nam11-config"'
+    test_vars_overrides:
+      instance_config_name: '"custom-tf-test-nam11-config"'
+# Sweeper skipped as this resource has customized deletion.
+skip_sweeper: true
```

```suggestion
skip_sweeper: true
exclude_tgc: true
```
Skip TGC for now - it's not a blocker